packages/amplify_flutter/lib/method_channel_amplify.dart

```diff
@@ -22,13 +22,19 @@ class MethodChannelAmplify extends AmplifyClass {
   MethodChannelAmplify() : super.protected();
 
   @override
-  Future<bool?> _configurePlatforms(String version, String configuration) {
-    return _channel.invokeMethod<bool>(
+  Future<bool?> _configurePlatforms(
+      String version, String configuration) async {
+    final configured = await _channel.invokeMethod<bool>(
       'configure',
       <String, Object>{
         'version': version,
         'configuration': configuration,
       },
     );
+    if (configured ?? false) {
```

Comment:
how do we know this statement is true for analytics plugin case?

Reply:
It's the result of this call here. It will always be true, right now.

https://github.com/aws-amplify/amplify-flutter/blob/ed16e42a3ab802e20ecdaeb4eb325cae704ce60c/packages/amplify_flutter/android/src/main/kotlin/com/amazonaws/amplify/Amplify.kt#L121